bot/exts/holidays/halloween/scarymovie.py

```diff
@@ -21,6 +21,13 @@ async def random_movie(self, ctx: commands.Context) -> None:
         """Randomly select a scary movie and display information about it."""
         async with ctx.typing():
             selection = await self.select_movie()
+            if not selection:
+                await ctx.send(embed=Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description=":warning: Failed to select a movie from the API",
+                    color=Colours.soft_red
+                ))
```

Comment:
It seems like there's an APIError class that can be raised which is handled by the error handler. I think it would fit here.

Reply:
The APIError exception (as per the docstring) is for when "an external API returns an error message". Since this isn't an error returned by the API, I didn't opt to use the exception. 

If we do want to use it, what should the `status_code` argument be? 200? The [error handling](https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/core/error_handler.py#L127-L134) for it also seems pretty bare (doesn't actually use the `.error_msg` or `.status_code`).

Reply:
I see. Nevermind then.